### PR TITLE
Transition to aggregate verifiier

### PR DESCRIPTION
**Pull request by biafra23**
Handles #

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -32,8 +32,10 @@ pipeline {
         WALLETCONNECT_PROJECT_ID_PROD = credentials('WALLETCONNECT_PROJECT_ID_PROD')
 	WEB3AUTH_GOOGLE_CLIENT_ID_PROD = credentials('WEB3AUTH_GOOGLE_CLIENT_ID_PROD')
 	WEB3AUTH_GOOGLE_CLIENT_ID_STAGING = credentials('WEB3AUTH_GOOGLE_CLIENT_ID_STAGING')
-	WEB3AUTH_GOOGLE_VERIFIER_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_PROD')
-	WEB3AUTH_GOOGLE_VERIFIER_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_STAGING')
+	WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_PROD')
+	WEB3AUTH_GOOGLE_VERIFIER_SUB_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_SUB_PROD')
+	WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_STAGING')
+	WEB3AUTH_GOOGLE_VERIFIER_SUB_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_SUB_STAGING')
```
Comment (biafra23) [inline Jenkinsfile]: We need these four as jenkins credentials

**Review by MouazAlzahabi — APPROVED** [review]

**Comment by MouazAlzahabi** [comment]
LGTM, I haven't tested it but looks like there is problem with Jenkins ?

File: Jenkinsfile (hunk 1 above)
Comment (DmitryBespalov) [reply Jenkinsfile]: please check that those credentials are set in the ci

Comment (biafra23) [reply Jenkinsfile]: I will add them to Jenkins

**Pull request merged**